InstagramClone/View Controllers/ComposeViewController.m

```diff
@@ -0,0 +1,81 @@
+//
+//  ComposeViewController.m
+//  InstagramClone
+//
+//  Created by Angelina Zhang on 6/27/22.
+//
+
+#import "ComposeViewController.h"
+#import "ComposeView.h"
+
+@interface ComposeViewController () <UIImagePickerControllerDelegate, UINavigationControllerDelegate, UITextViewDelegate>
+
+@property (strong, nonatomic) IBOutlet ComposeView *composeView;
+
+@end
+
+@implementation ComposeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.composeView.postCaption.text = @"Type here...";
+    self.composeView.postCaption.textColor = UIColor.lightGrayColor;
+    self.composeView.postCaption.delegate = self;
+    [self pushImagePickerController];
+}
+
+- (void)pushImagePickerController {
```

Comment:
Push is a funny word for this. I think 'show' or 'present' make more sense. Push implies it's being pushed onto a navigation stack? Or maybe it presents it modally so it's not in one of the navigation stacks?